web/src/components/index.js

```diff
@@ -0,0 +1,28 @@
+import WilcoxonPlotTile from './vis/WilcoxonPlotTile.vue';
+import AnovaTableTile from './vis/AnovaTableTile.vue';
+import { wilcoxon_zero_methods, wilcoxon_alternatives, plot_types } from '../utils/constants';
+
+export const analyses = [
+  {
+    path: 'wilcoxon', // same as key
+    name: 'Wilcoxon signed-rank test',
+    shortName: 'Wilcoxon Test',
+    description: 'The Wilcoxon signed-rank test tests the null hypothesis that two related paired samples come from the same distribution. In particular, it tests whether the distribution of the differences x - y is symmetric about zero. It is a non-parametric version of the paired T-test.',
+    component: WilcoxonPlotTile,
+    args: {
+      zero_method: wilcoxon_zero_methods[0].value,
+      alternative: wilcoxon_alternatives[0].value,
+    },
+    type: plot_types.ANALYSIS,
```

Comment:
isn't it always this plot type?

Reply:
No.  there's also plot_types.TRANSFORM.   Distinguishes which API to call.  I actually don't prefer this, I think it would be better to have a single URI for all plot types.

* `/csv/<id>/plot/pca`
* `/csv/<id>/analysis/anova`

These could both go under the `analysis` route IMO.

Reply:
I meant more why not specifying the `type: plot_types.ANALYSIS` when you mix it in the `plotDefaults`. There is no need that this is part of this analysis list

Reply:
Any new item could be added to the list here and may invoke a different endpoint to get the necessary data.  `data, valid, loading` are all safe to define during the mixin loop because they are properties managed by the vuex state.  type is a constant value intrinsic to the object declared in index.js